howtos/how-to-create-a-poll/public/index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>TalkJS tutorial test</title>
```

Comment:
Nitpick: I'd change the title to something like "TalkJS Polls Sample"

Reply:
Good catch!